uludag-universitesi-sosyal-bilimler-enstitusu-full-note.csl

```diff
@@ -1378,7 +1386,7 @@
       </choose>
     </layout>
   </citation>
-  <bibliography delimiter-precedes-et-al="never" delimiter-precedes-last="always" et-al-min="4" et-al-use-first="1" subsequent-author-substitute="&#8212;&#8212;&#8212;" entry-spacing="0" hanging-indent="true">
+  <bibliography delimiter-precedes-et-al="never" delimiter-precedes-last="always" et-al-min="4" et-al-use-first="1" subsequent-author-substitute="———" entry-spacing="0" hanging-indent="true">
```

Comment:
(@adam3smith, FYI, I never figured out how to escape em-dashes in the visual editor; I usually run most PRs manually through to https://formatter.citationstyles.org/ to fix any indenting issues and escape these characters)